pydicom/data/__init__.py

```diff
@@ -9,7 +9,7 @@
 __all__ = [
     'fetch_data_files',
     'get_charset_files',
-    'get_palette_files'
+    'get_palette_files',
```

Comment:
This was actually a bug...

Reply:
Yes, I was surprised to see LGTM find an actual bug in addition to warnings and recommendations.